src/main/java/edu/brown/cs/student/main/Main.java

```diff
@@ -40,23 +47,19 @@ public static void main(String[] args) {
 
   private String[] args;
 
+  // stars array: [[starID: String, starName: String, X: String, Y: String, Z: String]]
```

Comment:
This code is very readable. I would not have understood the purpose of these data structures without the comments.